composer.json

```diff
@@ -39,5 +41,10 @@
 	},
 	"autoload-dev": {
 		"classmap": ["tests/"]
+	},
+	"config": {
+		"platform": {
+			"ext-mongo": "1.6.16"
```

Comment:
What's this for?

Reply:
`doctrine/mongodb-odm` requires `ext-mongo`, this tells composer that its installed, even though it isnt, and allows odm to be installed

Reply:
The other option is to run `composer install` with `--ignore-platform-reqs`, if you think thats better @ondrejmirtes ?

Reply:
We can leave this in :)